Misc/NEWS.d/next/Core and Builtins/2020-06-17-10-27-17.bpo-40636.MYaCIe.rst

```diff
@@ -0,0 +1,3 @@
+:func:`zip` now supports :pep:`618`'s ``strict`` parameter, which raises a
+:exc:`ValueError` if the arguments are exhausted at differing lengths.
```

Comment:
Pretty sure "differing" is correct here (I think a strict reading of "different" may mean that *no two lengths* were the same).

I'll just change it anyways, since "different" probably makes more sense to more people.